.cloudbuild/ci/build.yaml

```diff
@@ -18,17 +18,17 @@ steps:
       - "go run ./cmd/checkout -w=/workspace"
 
   - name: gcr.io/cloud-builders/docker
-    id: type-check-and-lint
+    id: prettier-check-and-lint
```

Comment:
I think this step and the underlying npm script can be renamed to just `lint` – it can run both eslint and prettier check. `type-check-and-lint` is used only in CI, I added it in #874.

Reply:
I'll rename